client/model.go

```diff
@@ -3,11 +3,27 @@ package client
 import "encoding/json"
 
 type Organization struct {
-	Id           string `json:"id"`
-	Name         string `json:"name"`
-	CreatedBy    string `json:"createdBy"`
-	Role         string `json:"role"`
-	IsSelfHosted bool   `json:"isSelfHosted"`
+	Id                                  string  `json:"id"`
+	Name                                string  `json:"name"`
+	MaxTtl                              *string `json:"maxTtl"`
+	DefaultTtl                          *string `json:"defaultTtl"`
+	DoNotReportSkippedStatusChecks      bool    `json:"doNotReportSkippedStatusChecks"`
+	DoNotConsiderMergeCommitsForPrPlans bool    `json:"doNotConsiderMergeCommitsForPrPlans"`
+	Description                         string  `json:"description"`
+	PhotoUrl                            string  `json:"photoUrl"`
+	CreatedBy                           string  `json:"createdBy"`
+	CreatedAt                           string  `json:"createdAt"`
+	UpdatedAt                           string  `json:"updatedAt"`
+	Role                                string  `json:"role"`
+	IsSelfHosted                        bool    `json:"isSelfHosted"`
+	IsSelfHostedK8s                     bool    `json:"isSelfHostedK8s"`
+}
+
+type OrganizationPolicyUpdatePayload struct {
+	MaxTtl                              *string `json:"maxTtl"`
```

Comment:
null is actually meaningful in this case (infinite).
*string without omitempty is on purpose. (These fields are "required").